components/Window.vue

```diff
@@ -40,11 +40,11 @@
             </div>
         </div>
 
-        <div
-            v-for="(lines, index) in blocks"
-            :key="index"
-            :style="{ padding: `${settings.padding}px` }"
-        >
+        <div v-for="(lines, index) in blocks" :key="index" :style="{ padding: `${settings.padding}px` }">
+            <div
+                v-if="filenames.length > 1"
+                class="text-sm mb-2 text-gray-400 w-full text-right"
+            >{{ filenames[index] }}</div>
```

Comment:
Can we add an additional clause here to the `if` statement so the div isn't rendered if the filename is empty? Ex:

```vue
<div
  v-if="filenames.length > 1 && filenames[index]"
>
```

Once these two changes are made I'll merge this in 👍 

Thanks for your time and work! ❤️